### ISSUE TITLE
xemu: UI always accepts xinput even when controller is set to none

### ISSUE THREAD

**Issue by HadetTheUndying** [xemu-project/xemu#946]
### Bug Description

When plugging in Official 3rd Party Peripherals. (Steel Battallion Mega-Jockey-9000, DDR Dance pad, Madcatz Racing Wheel),they are always captured by xinput for UI navigation. Some of these peripherals cause the UI to scroll through options constantly. 

If I need to refile this under feature request I will but currently is does actually disrupt the usage of the application with the only option being to unplug the device.

### Expected Behavior

It would be nice to be able to bind a single device for UI navigation and not have every device accepted for it.

### xemu Version

Version:      0.7.17
Branch:       master
Commit:       66b57dca9c5c461c13cd44f977bd357677200be3
Date:         Fri May 20 05:42:48 PM UTC 2022

### System Information

CPU:          AMD Ryzen 7 2700X Eight-Core Processor         
OS Platform:  Linux
OS Version:   Void Linux
Manufacturer: AMD
GPU Model:    AMD Radeon RX Vega (vega10, LLVM 12.0.1, DRM 3.44, 5.16.20_1)
Driver:       4.6 (Core Profile) Mesa 22.2.0-devel (git-7938ce4af3)
Shader:       4.60

### Additional Context

This interferes with testing and development in certain situations as well as it also triggers the UI interaction in the VIdeo, Audio and Monitor UI elements and can cause them to be randomly jumped to

Video since @mborgerson confirmed this is not supposed to be happening:

https://user-images.githubusercontent.com/28496334/169624897-46810c0b-3fd7-4d82-a305-913b64d61bb5.mp4

.

**Comment by mborgerson** [comment]
Oh, yeah actually this was intentional. I thought you were talking about the controller test GUI being moved without being bound. The idea for this was that you could control the UI without having the controller bound, and bind the controller for instance. This can be made an option.

**Comment by HadetTheUndying** [comment]
Yeah I figured we might be misunderstanding each other. It's a very specific usecase as is. Only happens with my Steel Battalion controller and DDR pad.(The constant scrolling that is.) would be nice to have the option though. Thanks for checking it out.

**Comment by UrUsernameHere** [comment]
Since the emulator won't stop reopening the controller overlay while the controller is plugged which is another issue, this is a problem for me, forcing me to unplug the controller even when the input device is set to Keyboard.

**Comment by Triticum0** [comment]
@HadetTheUndying is this still an issue xemu now support ls sdl3 so might want to retest.
